packages/manager/modules/pci/src/projects/project/public-gateways/onboarding/onboarding.service.js

```diff
@@ -0,0 +1 @@
+export default class publigGatewaysOnboardingServiceClass {}
```

Comment:
Don't implement something which is not required.

Reply:
Removed as it was not required.